src/modules/capabilities.json

```diff
@@ -0,0 +1,718 @@
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": false
+                    }
+                }
+            ]
+        },
+        {
+            "name": "granted",
+            "summary": "Returns whether the current App has a user grant for passed capability and role.",
+            "params": [
+                {
+                    "name": "capability",
+                    "required": true,
+                    "schema": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Capability"
+                    }
+                },
+                {
+                    "name": "role",
+                    "schema": {
+                        "$ref": "https://meta.comcast.com/firebolt/capabilities#/definitions/Role",
+                        "default": "use"
+                    }
+                }
+            ],
+            "result": {
+                "name": "granted",
+                "summary": "Whether or not app is granted to use the given capability and the role",
+                "schema": {
+                    "type": "boolean"
+                }
+            },
+            "examples": [
+                {
+                    "name": "Default capabilities without grants.",
+                    "params": [
+                        {
+                            "name": "capability",
+                            "value": "xrn:firebolt:capability:keyboard"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": true
+                    }
+                },
+                {
+                    "name": "Get Postal code without grants.",
+                    "params": [
+                        {
+                            "name": "capability",
+                            "value": "xrn:firebolt:capability:localization:postalCode"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": false
+                    }
+                },
+                {
+                    "name": "Get Postal code with grants.",
+                    "params": [
+                        {
+                            "name": "capability",
+                            "value": "xrn:firebolt:capability:localization:postalCode"
+                        }
+                    ],
+                    "result": {
+                        "name": "Default Result",
+                        "value": true
+                    }
+                }
+            ]
+        },
+        {
+            "name": "info",
+            "summary": "Returns an array of CapabilityInfo objects for the passed in capabilities.",
+            "params": [
+                {
+                    "name": "role",
```

Comment:
I know it's in the spec this way, but I'm wondering if we should change the order of these params to match the other methods?

Reply:
I'm also thinking we could just drop the `role` param and update CapabilityInfo to have:

```
{
  supported; true,
  available: true,
  use: {
    permitted: true,
    granted: true
  },
  manage: {
    permitted: true,
    granted: true
  },
  provide: {
    permitted: true,
    granted: true
  }
}
```

Thoughts?

Reply:
Which capabilities can have multiple roles(use + manage, + provide)? 
Based on my prior discussions with @kpears201 each capability will have one or the other. Getters are mostly use, Setters are manage and providers are usually manifest/app driven.

If capabilities do not have multi faceted roles feel this will add default nodes and expand the response size.

Reply:
All capabilities have all three roles. Although `provide` is locked down in some instances where hardware is required.

It's true that a given app might only have one role, but it's not uncommon to have two, for example the settings app will likely have manage & use for several capabilities.

Reply:
yeah I like this idea, this API just aggregates all the roles in the response.

Reply:
@satlead might be a mix up. As Jeremy said, each capability has all 3 roles. I think maybe you are thinking that each method only has one role?

Reply:
https://github.com/rdkcentral/firebolt-schemas/pull/21

Reply:
https://github.com/rdkcentral/firebolt-schemas/pull/23